ilastik/workflows/edgeTrainingWithMulticut/edgeTrainingWithMulticutWorkflow.py

```diff
@@ -254,6 +254,14 @@ def handle_new_superpixels(*args):
         opEdgeTrainingWithMulticut.GroundtruthSegmentation.connect(opGroundtruthCache.Output)
         opEdgeTrainingWithMulticut.WatershedSelectedInput.connect(opWsdt.SelectedInput)
 
+        def _invalidate_cache_on_sp_change(*args, **kwargs):
+            op = opEdgeTrainingWithMulticut
+            print("Setting the cache dirty!", args, kwargs)
+            opEdgeTrainingWithMulticut.clear_caches(op.current_view_index())
+
+        opSuperpixelsSelect.Output.notifyDirty(_invalidate_cache_on_sp_change)
+        opSuperpixelsSelect.Output.notifyDirty(lambda *x, **xx: print("dirty"))
```

Comment:
Isn't `_invalidate_cache_on_sp_change` already doing the same thing?

Reply:
sorry that was leftover code, too.